run-custom-mao.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+
+# This script needs an machine-api-operator image as an input parameter
+# For example: ./run-custom-mao.sh quay.io/sdasu/machine-api-operator:sdasu-fix
+export SCRIPTS_DIR=$(pwd)
+export CUSTOM_MAO_BUILD=$1
+
+kubectl get secrets --namespace openshift-machine-api | grep mariadb
```

Comment:
left over debug?-

Reply:
Added this and "echo $REPLACEMENT_TEXT" so that the CI console logs will have this information that will help if there are any potential issues. For example, right now this command will show that "mariadb-password" exists. When this script is run as part of changes to remove the mariadb-password creation in dev-scripts, this should show that no secrets with "mariadb" exist. I think it has value in the short term, but can be removed later.